core/src/main/resources/jenkins/security/ApiTokenProperty/resources.js

```diff
@@ -61,7 +61,9 @@ function revokeToken(anchorRevoke) {
         fetch(targetUrl, {
           body: new URLSearchParams({ tokenUuid: tokenUuid }),
           method: "post",
-          headers: crumb.wrap({}),
+          headers: crumb.wrap({
+            "Content-Type": "application/x-www-form-urlencoded",
+          }),
```

Comment:
Not strictly needed, since the body is a `URLSearchParams` object, but done to create a consistent coding style within this repository, especially given that in some other places we need to use our own equivalent to `URLSearchParams` to satisfy HtmlUnit.